src/node/rpc/endpoint_registry.h

```diff
@@ -604,23 +604,6 @@ namespace ccf
      */
     void install(Endpoint& endpoint)
     {
-      if (endpoint.authn_policies.empty())
```

Comment:
The rest of the fix was mostly duplicated in #2048, but this was forgotten there, so scrubbing it here.

Reply:
Thanks, was just about to open another PR for this one.